packages/sdk/contractkit/package.json

```diff
@@ -21,7 +21,7 @@
     "prepublishOnly": "yarn build",
     "docs": "typedoc",
     "test": "jest --runInBand --forceExit",
-    "lint": "tslint -c tslint.json --project ."
+    "lint": "yarn run --top-level eslint -c .eslintrc.js --ext .ts src"
```

Comment:
i think the --ext .ts and src could be specified in the eslintrc no?

Reply:
maybe